test/common/testlib.py

```diff
@@ -1801,27 +1801,14 @@ def check_journal_messages(self, machine=None):
                 r"File .*asyncio/events.py.*",
                 r"self._context.run\(self._callback, \*self._args\)",
                 r"File .*cockpit/transports.py.* _read_ready",
-                r"data = os.read\(self._in_fd, _Transport.BLOCK_SIZE\)",
-                r"ConnectionResetError: \[Errno 104\] Connection reset by peer"]
+                r"data = os.read\(self._in_fd, _Transport.BLOCK_SIZE\)"]
```

Comment:
*mumbles something about trailing commas*

Reply:
*mumbles agreement, but is too lazy to re-push/re-run again*

We need to figure out the remaining issues anyway, at least this won't live forever..